ddl/column.go

```diff
@@ -32,48 +32,63 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// adjustColumnInfo is used to set the correct position of column info.
+// TODO: Index can't cover the add column with offset now, we may check this later.
```

Comment:
Can not understand the TODO.

Reply:
Just move it from L66, it means to adjust the index column offset, need a double check.

Reply:
We only need to do it when we support adding column and index at the same time.

Reply:
How about:
```
TODO: this implementation doesn't support adding column and index at the same time.
```

Reply:
In this function, it updates the offsets of the index columns. I think it is not about adding an index.

Reply:
Make it more readable, PTAL @coocood .